tests/kernel/boot_ltp.pm

```diff
@@ -41,6 +40,25 @@ sub run {
     }
 
     select_serial_terminal;
+}
+
+sub run {
+    my ($self) = @_;
+    my $cmd_file = get_var('LTP_COMMAND_FILE') || '';
+
+    boot_system($self);
+
+    my $ltp_selinux = get_var('LTP_SELINUX');
+    if (defined($ltp_selinux)) {
+        if ($ltp_selinux eq 'disabled') {
+            assert_script_run("sed -i -e 's|^SELINUX=.*|SELINUX=disabled|g' /etc/selinux/config");
+            power_action('reboot', textmode => 1);
+            boot_system($self);
+        } else {
+            script_run("setenforce ${ltp_selinux}");
```

Comment:
@czerw , Don't we need to reboot if we are changing from `disabled` to `enabled` ? or will we not encounter that scenario as by default it'll be enabled?

Reply:
I didn't count with that scenario, as we don't have it by default, but i can add option to enable SELinux if, is disabled.

Reply:
I decided to support disabling or setting permissive mode for now. We don't need to enable it for now and it would add more complexity.